<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisNetAppClass</class>
 <widget class="QMainWindow" name="VisNetAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisNetApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="widget_6" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <item row="4" column="1">
              <widget class="QPushButton" name="pushButton_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>停止</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>节点特征选择</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QPushButton" name="pushButton_6">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>重新运行</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>调试按钮</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" rowspan="2" colspan="2">
              <widget class="QCheckBox" name="checkBox_4">
               <property name="text">
                <string>节点特征1</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="pushButton_4">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>运行</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0" rowspan="2">
              <widget class="QCheckBox" name="checkBox_6">
               <property name="text">
                <string>节点特征3</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" rowspan="2">
              <widget class="QCheckBox" name="checkBox_5">
               <property name="text">
                <string>节点特征2</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <item>
             <property name="text">
              <string>已添加的文件：</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="net">
          <attribute name="title">
           <string>net</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="tablenet"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="name">
          <attribute name="title">
           <string>name</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tablename"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="feature1">
          <attribute name="title">
           <string>feature1</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="tablefeature1"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="feature2">
          <attribute name="title">
           <string>feature2</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableWidget" name="tablefeature2"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="feature3">
          <attribute name="title">
           <string>feature3</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTableWidget" name="tableWidget_4"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_add_net"/>
    <addaction name="action_add_name"/>
    <addaction name="action_add_feature1"/>
    <addaction name="action_add_feature2"/>
    <addaction name="action_add_feature3"/>
   </widget>
   <widget class="QMenu" name="menu_set">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_set_ndsize"/>
    <addaction name="action_set_ndcolor"/>
    <addaction name="action_set_link"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_set"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_add_net">
   <property name="text">
    <string>添加网络文件</string>
   </property>
  </action>
  <action name="action_add_name">
   <property name="text">
    <string>添加节点名称映射表</string>
   </property>
  </action>
  <action name="action_add_feature1">
   <property name="text">
    <string>添加节点特征文件1</string>
   </property>
  </action>
  <action name="action_add_feature2">
   <property name="text">
    <string>添加节点特征文件2</string>
   </property>
  </action>
  <action name="action_add_feature3">
   <property name="text">
    <string>添加节点特征文件3</string>
   </property>
  </action>
  <action name="action_set_ndsize">
   <property name="text">
    <string>设置节点大小</string>
   </property>
  </action>
  <action name="action_set_ndcolor">
   <property name="text">
    <string>设置节点颜色</string>
   </property>
  </action>
  <action name="action_set_link">
   <property name="text">
    <string>设置连接粗细</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VisNetApp.qrc"/>
 </resources>
 <connections/>
</ui>
